src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -89,6 +120,28 @@ export default function withAnnotations<P extends object>(
             annotator.emit('annotations_remove', id);
         };
 
+        createElementsError = (error: ElementsError | Error): ElementsError => {
+            const { message = '' } = error;
+            const origin = 'origin' in error && error.origin ? error.origin : ORIGIN_CONTENT_PREVIEW;
```

Comment:
It seems like the origin would always be `ORIGIN_CONTENT_PREVIEW`, since this HOC wraps that element.